### DISCUSSION TITLE
Equivalent of WPF DependencyObject.GetLocalValueEnumerator

### DISCUSSION THREAD

**Discussion by workgroupengineering** [unoplatform/uno#6380] [Q&A] [answered]
Hi,
there is equivalent  of WPF [DependencyObject.GetLocalValueEnumerator](https://docs.microsoft.com/en-us/dotnet/api/system.windows.dependencyobject.getlocalvalueenumerator?view=net-5.0)?

**Comment by davidjohnoliver** [comment, marked as answer]
I don't think there's any direct equivalent of that method in the WinUI/UWP API. You'd have to call `DependencyObject.ReadLocalValue()` for each `DependencyProperty`, and check whether it returned `UnsetValue`.

**Reply by workgroupengineering** [reply]
thanks I try
